tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -108,23 +108,23 @@ steps:
                   exit 1
                 fi
             else
-              feedId=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
+              feedName=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
 
-              if [[ $release == "release" && $feedId == "build" ]]; then
+              if [[ "$(release)" == "release" && $feedName == "build" ]]; then
```

Comment:
This is the primary problem: `release` is an ADO variable, not a variable within the bash script. As such, this comparison never succeeded and we never ran any of this logic.